xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/LiveTableRequestHandler.java

```diff
@@ -242,7 +242,16 @@ private void addFilterRequestParameters(Filter filter, Map<String, String[]> req
                 .map(constraint -> constraint == null ? null : constraint.getOperator())
                 .map(operator -> MATCH_TYPE.getOrDefault(operator, StringUtils.defaultString(operator)))
                 .collect(Collectors.toList());
-            requestParams.put(filter.getProperty() + "_match", matchType.toArray(new String[matchType.size()]));
+            requestParams.put(filter.getProperty() + "_match", matchType.toArray(new String[values.size()]));
```

Comment:
``values`` probably has the same size as ``matchType`` but it feels more natural to use the later.